Project.toml

```diff
@@ -1,6 +1,12 @@
 name = "ClimaComms"
 uuid = "3a4d1b5c-c61d-41fd-a00a-5873ba7a1b0d"
-authors = ["CliMA Contributors <clima-software@caltech.edu>"]
+authors = [
+    "Kiran Pamnany <clima-software@caltech.edu>",
+    "Simon Byrne <simonbyrne@caltech.edu>",
+    "Charles Kawczynski <charliek@caltech.edu>",
+    "Sriharsha Kandala <clima-software@caltech.edu>",
+    "Jake Bolewski <clima-software@caltech.edu>",
```

Comment:
@jakebolewski I'm adding authors to the repo: would you like your personal email here (and on other repos)? or happy to leave the clima one?

Reply:
I'll merge it as-is for now. If you want to change it later, please let me know.